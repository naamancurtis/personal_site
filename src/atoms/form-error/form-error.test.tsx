import React from 'react';
import { shallow } from 'enzyme';
import FormError from './form-error.component';

describe('<FormError />', () => {
  it('expect to render component', () => {
    const wrapper = shallow(<FormError message={'test'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
