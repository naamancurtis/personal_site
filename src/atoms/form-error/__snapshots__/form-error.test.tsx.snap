// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormError /> expect to render component 1`] = `
<styled.div>
  <FontAwesomeIcon
    border={false}
    className=""
    fixedWidth={false}
    flip={null}
    icon={
      Array [
        "fas",
        "exclamation-circle",
      ]
    }
    inverse={false}
    listItem={false}
    mask={null}
    pull={null}
    pulse={false}
    rotation={null}
    size={null}
    spin={false}
    swapOpacity={false}
    symbol={false}
    title=""
    transform={null}
  />
  <styled.span>
    test
  </styled.span>
</styled.div>
`;
